bip-musig2.mediawiki

```diff
@@ -522,23 +526,19 @@ Note that the aggregate public key and list of tweaks are inputs to partial sign
 
 === Dealing with Infinity in Nonce Aggregation ===
 
-If it happens that ''is_infinite(R'<sub>i</sub>)'' inside ''[[#NonceAgg infinity|NonceAgg]]''  there is at least one dishonest signer (except with negligible probability).
-If we fail here, we will never be able to determine who it is.
-Therefore, we continue so that the culprit is revealed when collecting and verifying partial signatures.
+If it happens that ''is_infinite(R')'' inside ''[[#NonceAgg infinity|GetSessionValues]]'', either the nonce aggregator is dishonest or there is at least one dishonest signer (except with negligible probability).
```

Comment:
If the nonce aggregator is dishonest, then strictly speaking, it doesn't make sense to talk about it running the honest "GetSessionValues" algorithm, so I think it's better to drop the sentence.

Maybe a better way is to rephrase it using the aggnonce:
```suggestion
If it happens ''aggnonce = bytes(33,0) || bytes(33,0)'' for the aggregate nonce returned by the signature aggregator, either the nonce aggregator is dishonest or there is at least one dishonest signer (except with negligible probability).
```

Reply:
I applied your suggestion. Do you still propose to drop a sentence?